ui/src/components/Dialog.tsx

```diff
@@ -53,6 +55,11 @@ export const DialogContent = React.forwardRef<
             <X16Icon className="h-4 w-4" />
           </DialogPrimitive.Close>
         )}
+        {showClose && appModal && (
+          <DialogPrimitive.Close className="icon-button absolute top-8 right-6 bg-white">
+            <X16Icon className="h-4 w-4" />
+          </DialogPrimitive.Close>
+        )}
```

Comment:
Wondering if maybe we shouldn't just get rid of the booleans and just make this a slot so whoever calling can just put the close where they want

Reply:
Agreed, but outside of scope here, right?